scripts/install-aws.sh

```diff
@@ -1,14 +1,42 @@
 #!/usr/bin/env bash
-echo "===== Starting installing AWS-CNI ========="
-sed -i s/__VETHPREFIX__/${AWS_VPC_K8S_CNI_VETHPREFIX:-"eni"}/g /app/10-aws.conflist
-cp /app/aws-cni /host/opt/cni/bin/
-cp /app/portmap /host/opt/cni/bin/
-cp /app/aws-cni-support.sh /host/opt/cni/bin/
-cp /app/10-aws.conflist /host/etc/cni/net.d/
-
-if [[ -f /host/etc/cni/net.d/aws.conf ]]; then
+
+grpcHealthCheck () {
+  /app/grpc_health_probe -addr 127.0.0.1:50051
```

Comment:
Where is the `grpc_health_probe` coming from?

Wouldn't Dockerfile.release need something like this?
```
RUN GRPC_HEALTH_PROBE_VERSION=v0.3.0 && \
    wget -qO/app/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
    chmod +x /app/grpc_health_probe
```